src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  Component,
+  Inject,
+  Input,
+  OnChanges,
+  SimpleChanges,
+} from "@angular/core";
+import { EntityConstructor } from "../../../../entity/model/entity";
+import {
+  MAT_DIALOG_DATA,
+  MatDialogModule,
+  MatDialogRef,
+} from "@angular/material/dialog";
+import { MatButtonModule } from "@angular/material/button";
+import { DialogCloseComponent } from "../../../../common-components/dialog-close/dialog-close.component";
+import { MatInputModule } from "@angular/material/input";
+import { ErrorHintComponent } from "../../../../common-components/error-hint/error-hint.component";
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  FormsModule,
+  ReactiveFormsModule,
+  Validators,
+} from "@angular/forms";
+import { EntitySchemaField } from "../../../../entity/schema/entity-schema-field";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { AdminEntityService } from "../../../admin-entity.service";
+import { v4 as uuid } from "uuid";
+
+@Component({
+  selector: "app-admin-entity-text",
+  standalone: true,
+  imports: [
+    MatDialogModule,
+    MatButtonModule,
+    DialogCloseComponent,
+    MatInputModule,
+    ErrorHintComponent,
+    FormsModule,
+    ReactiveFormsModule,
+    FontAwesomeModule,
+    MatTooltipModule,
+  ],
+  templateUrl: "./admin-entity-text.component.html",
+  styleUrl: "./admin-entity-text.component.scss",
+})
+export class AdminEntityTextComponent implements OnChanges {
```

Comment:
Can you rename this to "AdminEditDescriptionOnlyFieldComponent" please? (aligning 1-to-1 with the field schema name)

Reply:
sure let me make this adjustments

Reply:
Updated the name of `AdminEntityTextComponent` to `AdminEditDescriptionOnlyFieldComponent`, ensuring consistency across folder names and selections.